core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -873,8 +873,13 @@ public enum Property {
       "The listening port for the garbage collector's monitor service", "1.3.5"),
   GC_DELETE_THREADS("gc.threads.delete", "16", PropertyType.COUNT,
       "The number of threads used to delete RFiles and write-ahead logs", "1.3.5"),
+  @Deprecated(since = "2.1.1", forRemoval = true)
   GC_TRASH_IGNORE("gc.trash.ignore", "false", PropertyType.BOOLEAN,
       "Do not use the Trash, even if it is configured.", "1.5.0"),
+  GC_USE_TRASH("gc.use.trash", "true", PropertyType.STRING,
+      "Moves a file to the Trash (if available). Valid values are true, false, and bulk_imports_only."
+          + " Mutually exclusive with gc.trash.ignore.",
+      "2.1.1"),
```

Comment:
I like the fix of the negation. But, I'm wondering if there are use cases for use trash for `everything_except_bulk_imports`  or something different for WALs and RFiles. I am thinking we should have a more robust TrashPolicy SPI, where we give it everything we know about a file, and it returns "Trash" or "Delete". The default implementation would be "AlwaysTrash", but we could also provide a "NeverTrash" implementation, and anything else more complicated would be up to the user to provide.